src/main/java/com/vinylteam/vinyl/service/HasShopIdentifier.java

```diff
@@ -0,0 +1,7 @@
+package com.vinylteam.vinyl.service;
+
+public interface HasShopIdentifier {
+
+    long getShopId();
+
+}
```

Comment:
Can me put method 
long getShopId() to interface VinylParser? 
Isn`t interface HasShopIdentifier redundant?